src/SystemCommands-SourceCodeCommands/SycExtractTempCommand.class.st

```diff
@@ -49,6 +40,23 @@ SycExtractTempCommand >> defaultMenuItemName [
 	^ 'Extract temp'
 ]
 
+{ #category : 'execution' }
+SycExtractTempCommand >> execute [
+
+	ReExtractToTemporaryDriver new
+		extract: sourceNode sourceInterval
+		to: tempName
+		from: method selector
+		in: method origin;
+		model: RBBrowserEnvironment new;
+		scopes: toolContext refactoringScopes; runRefactoring
+]
```

Comment:
Here, you should not pass in model like this, it's a common mistake that bit me many times. You can find correct way of doing it in one of the drivers. Basically you do this: `self refactoringScopeOn: scopes first.`

So suggestion is to remove the line 51, and add something like this in driver constructor:
https://github.com/pharo-project/pharo/blob/f55a096d163d2ed5fbffb91bb5365477e1a9915e/src/Refactoring-UI/ReRemoveClassDriver.class.st#L124